experimental/Schemes/SimplifiedSpec.jl

```diff
@@ -171,6 +172,20 @@ function _flatten_open_subscheme(
   return _flatten_open_subscheme(W, C, iso=new_iso)
 end
 
+function _flatten_open_subscheme(
```

Comment:
Here I am not sure that I see the purpose of this function. Can you give me a hint?

Reply:
It's part of part I. See the discussion there.